scripts/dependencies.js

```diff
@@ -0,0 +1,80 @@
+// eslint-disable-next-line import/no-cycle
+import {
+  decorateBlock,
+  fetchPlaceholders,
+  loadBlock,
+  loadScript,
+  sendAnalyticsPageEvent,
+} from './scripts.js';
+
+const placeholders = await fetchPlaceholders();
+const isProd = window.location.hostname.endsWith(placeholders.hostname);
+
+/* setup cookie preferences */
+function getCookie(cookieName) {
+  const name = `${cookieName}=`;
+  const decodedCookie = decodeURIComponent(document.cookie);
+  const split = decodedCookie.split(';');
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < split.length; i++) {
+    let c = split[i];
+    while (c.charAt(0) === ' ') c = c.substring(1);
+    if (c.indexOf(name) === 0) return c.substring(name.length, c.length);
+  }
+  return null;
+}
+
+async function OptanonWrapper() {
+  const geoInfo = window.Optanon.getGeolocationData();
+  Object.keys(geoInfo).forEach((key) => {
+    const cookieName = `PGAT_${key.charAt(0).toUpperCase() + key.slice(1)}`;
+    const cookie = getCookie(cookieName);
+    if (!cookie || cookie !== geoInfo[key]) document.cookie = `${cookieName}=${geoInfo[key]}`;
+  });
+
+  const prevOptIn = localStorage.getItem('OptIn_PreviousPermissions');
+  if (prevOptIn) {
+    try {
+      const settings = JSON.parse(prevOptIn);
+      if (settings.tempImplied) {
+        localStorage.removeItem('OptIn_PreviousPermissions');
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error('OptIn_PreviousPermissions parse failed');
+    }
+  }
+  sendAnalyticsPageEvent();
```

Comment:
If you want, we can remove this function because we are not using it in GTM to fire page load events